### ISSUE TITLE
MioU is too low, the val_loss and val_MIoU curve are very weird.(deeplabv3+ and xception, Cityscapes)

### ISSUE THREAD

**Issue by XiaXinjie** [yassouali/pytorch-segmentation#119]
Thank you very much. I am not goot at english, but I have some questions and need some help.

the config.json:
{
    "name": "deeplabv3_plus",
    "n_gpu": 1,
    "use_synch_bn": false,

    "arch": {
        "type": "DeepLab",
        "args": {
            "backbone": "Xception",
            "freeze_bn": false,
            "freeze_backbone": false
        }
    },

    "train_loader": {
        "type": "CityScapes",
        "args":{
            "data_dir": "data/.....",
            "batch_size": 8,
            "base_size": 400,
            "crop_size": 380,
            "augment": true,
            "shuffle": true,
            "scale": true,
            "flip": true,
            "rotate": true,
            "blur": false,
            "split": "train_aug",
            "num_workers": 8
        }
    },

    "val_loader": {
        "type": "CityScapes",
        "args":{
            "data_dir": "data/......",
            "batch_size": 8,
            "crop_size": 480,
            "val": true,
            "split": "val",
            "num_workers": 4
        }
    },

    "optimizer": {
        "type": "SGD",
        "differential_lr": true,
        "args":{
            "lr": 0.01,
            "weight_decay": 1e-4,
            "momentum": 0.9
        }
    },

    "loss": "CrossEntropyLoss2d",
    "ignore_index": 255,
    "lr_scheduler": {
        "type": "Poly",
        "args": {}
    },

    "trainer": {
        "epochs": 100,
        "save_dir": "saved/",
        "save_period": 10,
  
        "monitor": "max Mean_IoU",
        "early_stop": 10,
        
        "tensorboard": true,
        "log_dir": "saved/runs",
        "log_per_iter": 20,

        "val": true,
        "val_per_epochs": 10
    }
According to https://github.com/yassouali/pytorch-segmentation/pull/110#issue-620001051, I replaced the deeplabv3_plus code and used the new Xception code. The results are showing as follow:

![Train_Loss](https://user-images.githubusercontent.com/87801544/126605852-67509d63-ec8f-4f62-8d24-6940582e9e6a.png)
![Train_MIoU](https://user-images.githubusercontent.com/87801544/126605932-b42400ee-0191-4c61-81e0-62d05473508a.png)
![Train_Pixel_Acc](https://user-images.githubusercontent.com/87801544/126605945-6d1e0a88-fa0a-42ab-a0e7-720d922e2beb.png)
![Val_Loss](https://user-images.githubusercontent.com/87801544/126605958-5dbccbb5-1de7-4358-a43b-90af25e119b8.png)
![Val_MIoU](https://user-images.githubusercontent.com/87801544/126605980-135242f2-85f4-41c2-b94b-077f30675d41.png)
![Val_Pixel_Acc](https://user-images.githubusercontent.com/87801544/126605996-5c39f19d-54c6-4e17-bd44-9d31a59b995c.png)
In this results, the Validation results have some strange，I am eager to konw what's wrong.

**Comment by XiaXinjie** [comment]
I am sorry, "split": "train" is corret.

**Comment by cm0561** [comment]
你好  可以加你联系方式吗   我最近在用这个代码 出现一样的问题   跑city数据集 时候 精度不是很理想  你解决问题了吗  我WV 18155759923

**Comment by yassouali** [comment]
can you try the newly added model [deeplabv3_xception.py](https://github.com/yassouali/pytorch-segmentation/blob/master/models/deeplabv3_xception.py)
